orm/src/main/java/org/hibernate/search/hcore/impl/HibernateSearchSessionFactoryObserver.java

```diff
@@ -93,8 +110,40 @@ public void sessionFactoryCreated(SessionFactory factory) {
 		}
 	}
 
+	private void startTriggerAsyncBackendService(SessionFactory factory) {
```

Comment:
I will have to take a look into why I did this the way I did it. Maybe it was related to me not getting a proper reference to the SessionFactory, but I don't know right now.